plugins/link/plugin.js

```diff
@@ -208,7 +218,7 @@
 
 	// Loads the parameters in a selected link to the link dialog fields.
 	var javascriptProtocolRegex = /^javascript:/,
-		emailRegex = /^mailto:([^?]+)(?:\?(.+))?$/,
+		emailRegex = /^(?:mailto)(?:(?!\?(subject|body)=).)+/i,
```

Comment:
There is one problem with this approach. Link dialog handles `subject` and `body` fields only, however there can be much more (even just custom query parameters). Such content can be simply loaded into CKEditor from external source and will be displayed incorrectly.

That being said, we have already discussed it with @Dumluregn F2F and extracted it to #3253 as a follow-up. And in case of invalid content (email address with special chars not encoded) loaded externally we can do nothing as there is no 100% accurate way to distinguish email address from query string. So this approach seems valid in this case.